frontend/src/components/tournament/MatchCard.js

```diff
@@ -55,8 +55,15 @@ const MatchCard = props => {
           <div
             class={`w-full flex justify-center flex-wrap bg-${color()}-100 text-${color()}-800 text-xs font-medium mr-2 px-2.5 py-0.5 rounded dark:bg-${color()}-900 dark:text-${color()}-300`}
           >
-            <h6 class="text-center w-full">Pool {props.match.pool.name}</h6>
-            <Show when={props.showSeed}>
+            <Show
```

Comment:
Can you add UI screenshots as well.. For both showSeed true and false.. for pools and brackets